.github/workflows/vmimage.yml

```diff
@@ -3,17 +3,21 @@ name: "VMImage CI"
 on:
   push:
     branches: [ "master", "103lts" ]
-    paths: 
+    paths:
       - 'avocado/utils/vmimage.py'
       - 'selftests/utils/vmimage.py'
+      - 'selftests/vmimage/**'
+      - 'avocado/plugins/runners/vmimage.py'
```

Comment:
IMO all changes in this file are related to the vmimage testing refactor in https://github.com/avocado-framework/avocado/commit/7937e42ad6470c96a077bfedf2a7cd9b31eeae21 , am I right? Therefore, I would move it to that commit.